src/main/java/seedu/lifeasier/commands/DisplayScheduleCommand.java

```diff
@@ -2,42 +2,53 @@
 
 import seedu.lifeasier.model.notes.NoteHistory;
 import seedu.lifeasier.model.notes.NoteList;
-import seedu.lifeasier.parser.Parser;
-import seedu.lifeasier.storage.FileStorage;
 import seedu.lifeasier.model.tasks.TaskHistory;
 import seedu.lifeasier.model.tasks.TaskList;
+import seedu.lifeasier.parser.Parser;
+import seedu.lifeasier.storage.FileStorage;
 import seedu.lifeasier.ui.ScheduleUi;
 import seedu.lifeasier.ui.Ui;
 
 import java.time.LocalDate;
+import java.time.temporal.ChronoUnit;
 
 public class DisplayScheduleCommand extends Command {
 
-    private boolean isDisplayWeek;
+    private String displayKeyword;
     private final ScheduleUi scheduleUi = new ScheduleUi();
 
-    private static final String NO_TASKS_TODAY_MESSAGE = "You have nothing on for today!";
-
     public DisplayScheduleCommand(String toDisplay) {
-        this.isDisplayWeek = toDisplay.equals("week");
+        this.displayKeyword = toDisplay;
     }
 
     @Override
     public void execute(Ui ui, NoteList notes, TaskList tasks, FileStorage storage, Parser parser,
                         NoteHistory noteHistory, TaskHistory taskHistory) {
         LocalDate currDate = LocalDate.now();
-        if (isDisplayWeek) {
-            scheduleUi.showHome(tasks);
-        } else {
-            int taskCountForToday = scheduleUi.getTaskCountForToday(tasks, currDate);
-
-            if (taskCountForToday > 0) {
-                System.out.println("Here is your schedule for today:");
-                scheduleUi.displayDaySchedule(currDate, tasks);
-            } else {
-                System.out.println(NO_TASKS_TODAY_MESSAGE);
-            }
+        switch (displayKeyword) {
+        case "week":
+            scheduleUi.showHome(tasks, ui);
+            break;
+        case "today":
+            displayScheduleFor(currDate, tasks, ui);
+            break;
+        case "tomorrow":
+            displayScheduleFor(currDate.plus(1, ChronoUnit.DAYS), tasks, ui);
+            break;
+        default:
+            ui.showInvalidDisplayKeyword();
```

Comment:
Can rename this to match Daniel's naming style
```suggestion
            ui.showInvalidDisplayKeywordError();
```